Auth/Tests/Auth.Platform.Android.Tests/Credentials/OIDCCredentialManagerTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using AeroGear.Mobile.Auth.Credentials;
+using NUnit.Framework;
+
+namespace Auth.Platform.Android.Tests.Credentials
+{
+    [TestFixture]
+    public class CredentialsTest
+    {
+        private OIDCCredentialManager CredStore;
+        private readonly string CredStoreKey = "authState";
+
+        [SetUp]
+        public void Setup()
+        {
+            CredStore = OIDCCredentialManager.Instance;
+        }
+
+
+        [TearDown]
+        public void Tear() { }
```

Comment:
Does this do anything?

Reply:
This was generated as part of the test class, fairly sure I can remove it. Thanks